src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -35,17 +39,16 @@ export const CsvUploaderContext = React.createContext<CsvUploaderContextType>(
 )
 
 /**
- * This is an arbitrary number that should be capped out at 1750.
- * For now, the number is set to 500 since there were significant performance issues
- * that had cropped up when writing 1750 lines on tools that took ~20s for a write query
- * to resolve
+ * This is an arbitrary number that should be capped out at 1750 due to max data sizes
  */
-const MAX_CHUNK_SIZE = 500
+const MAX_CHUNK_SIZE = 1750
```

Comment:
not denying that this will help alleviate the problem in tools, but changing a magic number to fix it is the wrong solution. there are no guarantees that this number works in both tools and production, and if it's determined by the deployment environment of the api, that also means there's an undocumented constant out there that can drastically affect the user experience, leaving this interface susceptible to being disabled as a side effect of a configuration file that has no direct connection to the interface.

if there's too much infrastructure required to make this constant derive it's value from the api option that alters it, at least providing some documentation here on why tools is different that local development and what mechanism affects that performance should help mitigate the tight coupling.